### PR TITLE
Added support for latest formatter-bundle

### PR DESCRIPTION
**Pull request by greg0ire**
I am targeting this branch, because this is BC. 

## Changelog

```markdown
### Added
- Added support for latest `formatter-bundle`
```

### CODE REVIEW

**Comment by greg0ire** [comment]
See https://github.com/sonata-project/SonataFormatterBundle/pull/363

**Comment by greg0ire** [comment]
It's ready!

File: tests/Controller/Api/PostControllerTest.php [hunk 1]
```diff
@@ -360,9 +362,17 @@ protected function createPostController($postManager = null, $commentManager = n
...
+            $formatterPool = $this->getPoolMock();
         }
 
         return new PostController($postManager, $commentManager, $mailer, $formFactory, $formatterPool);
     }
+
+    /**
+     * @return mixed Pool|PoolInterface
+     */
+    private function getPoolMock()
```
Comment (kunicmarko20) [inline tests/Controller/Api/PostControllerTest.php]: Add a next major tag to remove this, we will probably drop < formatter 4.x on next major

Comment (greg0ire) [reply tests/Controller/Api/PostControllerTest.php]: Done

**Comment by kunicmarko20** [comment]
Build is failing

**Comment by greg0ire** [comment]
> Build is failing

Will have to revert to the previous solution, it was fine, this one cannot work :sweat_smile:

**Pull request merged**